src/pip/_internal/index/collector.py

```diff
@@ -468,37 +527,39 @@ def _get_html_page(
             url += "/"
         url = urllib.parse.urljoin(url, "index.html")
         logger.debug(" file: URL is directory, getting %s", url)
+        # TODO: index.json?
```

Comment:
What is there to do here ?

Reply:
I expanded on the comment.

Basically, I don't think there is anything for this PR to do.

That section of code is where pip "fakes" a simple repository on the local filesystem by translating an url like ``file:///my-repo/`` to the file system path ``/my-repo/index.html``. There's no standard file extension for ``application/vnd.pypi.simple.v1+json``, and I think the question of how pip's specific extension to the simple repo to map it to the filesystem is nuanced enough that it deserves it's own PR.

I also think that most of the value is in the PR without that change.